src/main/java/com/dakar/dakar/controller/GraphQlController.java

```diff
@@ -27,6 +29,9 @@
     @Autowired
     private GraphQL graphQL;
 
+    // Logger
+    private static final Logger logger = LogManager.getLogger(GraphQlController.class);
```

Comment:
this should be done automatically by the @Slf4j annotation
If it's not the case, then we have a problem

Reply:
you are right